BazelExtensions/xcode_configuration_provider.bzl

```diff
@@ -117,7 +117,7 @@ def _extract_generated_sources(target, ctx):
         objc = target[ObjcInfo]
         files.append(depset(objc.direct_headers))
     trans_files = depset(transitive = files)
-    return [f for f in trans_files.to_list() if not f.is_source and not f.path.endswith("-Swift.h")]
```

Comment:
In pure xcode mode, doesn't this implicate bazel swift compilation as a dependency?

Reply:
Without this taking a dep on a fmw that is mostly objc but has .swift files too won't build 

Caught this by building one internal vertical that takes a dep on a fmw that is mostly objc but had one single .swift file in it 😅 

IIUC we do want to trigger swift compilation in that scenario right?

Reply:
> Without this taking a dep on a fmw that is mostly objc but has .swift files too won't build

Xcode should compile the swift files and generate the header before it compiles the objc? Let's sync up on this on vc